lib/deploy.js

```diff
@@ -56,7 +56,8 @@ Deploy.prototype.deploy_contracts = function(env) {
       gasPrice: contract.gasPrice
     });
 
-    contractAddress = contractObject["new"].apply(contractObject, contractParams).address;
+    contractObject["new"].apply(contractObject, contractParams);
+    contractAddress = undefined; // TODO;
```

Comment:
we can't get address synchronously right know. I just wanted mark this as `TODO`